src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9saktillos/K9SakTilLosAdapterTjeneste.kt

```diff
@@ -153,14 +151,33 @@ class K9SakTilLosAdapterTjeneste(
         TODO()
     }
 
+    private fun oppgaveSkalHaVentestatus(event: BehandlingProsessEventDto): Boolean {
+        val åpneAksjonspunkter = event.aksjonspunktTilstander.filter { aksjonspunktTilstand ->
+            aksjonspunktTilstand.status.erÅpentAksjonspunkt()
+        }
+        val oppgaveFeltVerdiDtos = mutableListOf<OppgaveFeltverdiDto>()
+
+        utledAvventerflagg(event.behandlingSteg, åpneAksjonspunkter = åpneAksjonspunkter, oppgaveFeltverdiDtos = oppgaveFeltVerdiDtos)
+
+        return !oppgaveFeltVerdiDtos.any { it.nøkkel == "avventerSaksbehandler" && it.verdi == "true" }
+    }
+
     private fun lagOppgaveDto(event: BehandlingProsessEventDto, forrigeOppgave: OppgaveV3?) =
         OppgaveDto(
             id = event.eksternId.toString(),
             versjon = event.eventTid.toString(),
             område = "K9",
             kildeområde = "K9",
             type = "k9sak",
-            status = event.aksjonspunktTilstander.lastOrNull()?.status?.kode ?: "OPPR", // TODO statuser må gås opp
+            status = if (event.aksjonspunktTilstander.any { aksjonspunktTilstandDto -> aksjonspunktTilstandDto.status.erÅpentAksjonspunkt() }) {
+                if (oppgaveSkalHaVentestatus(event)) {
+                    "VENTER"
+                } else {
+                    "AAPEN"
+                }
+            } else {
+                "LUKKET"
```

Comment:
Samme som for klage:
Mener å huske at du sa at man helt initelt ikke vil ha aksjonspunkt på behandlingen. Kanskje vi i så fall bør ha en sjekk mot behandlingsstatus for å unngå at behandlingen starter som lukket?